test/models/host_test.rb

```diff
@@ -3086,20 +3086,6 @@ def to_managed!
       host = Host.new(:name => "test-host", :hostgroup => hg)
       assert host.environment
     end
-
-    test 'should filter out unpermitted strong parameters' do
-      host = FactoryBot.build(:host, :managed, :with_hostgroup)
```

Comment:
why are we removing these tests? thanks

Reply:
They are no longer relevant - the method is not passed strong params, they are dealt with in controller.